### PR TITLE
Added index_column to display the position of the row in the total collection.

### PR DESCRIPTION
**Pull request by cadetstar**
This is a proposed solution to fulfill the Feature Request activeadmin/activeadmin#3693

It only extends IndexAsTable for now, but could be built upon to do the same in other index styles.

### CODE REVIEW

File: lib/active_admin/views/index_as_table.rb [hunk 1]
```diff
@@ -244,6 +254,12 @@ def selectable_column
           end
         end
 
+        def index_column
+          column '#', class: 'col-index', sortable: false do |resource|
+            @collection.offset_value + @collection.index(resource) + 1
```
Comment (timoschilling) [inline lib/active_admin/views/index_as_table.rb]: I wold like to see the `+1` configurable:
```ruby
def index_column(offset = 1)
```

Comment (timoschilling) [inline lib/active_admin/views/index_as_table.rb]: Then the developer can make a `0` or `1` based index

File: lib/active_admin/views/index_as_table.rb [hunk 2]
```diff
@@ -108,6 +108,16 @@ module Views
...
+    # In addition, you can insert the position of the row in the greater collection by using the index_column special command:
+    #
+    # ```ruby
+    # index do
+    #   selectable_column
+    #   index_column
+    #   column :title
+    # end
+    # ```
+    #
```
Comment (timoschilling) [inline lib/active_admin/views/index_as_table.rb]: Can you add the same to the docs please.

File: docs/3-index-pages/index-as-table.md [hunk 3]
```diff
@@ -55,8 +55,6 @@ index do
 end
 ```
 
-## Defining Actions
```
Comment (timoschilling) [inline docs/3-index-pages/index-as-table.md]: Why did you remove this headline?

Comment (cadetstar) [inline docs/3-index-pages/index-as-table.md]: That was removed when I ran rake yard, but I didn't change the source file in that location.  I can add it back, but it had already been removed from the source documentation.

**Comment by timoschilling** [comment]
please make a rebase on master
